packages/contracts/test/voting/token-voting.ts

```diff
@@ -107,6 +116,28 @@ describe('TokenVoting', function () {
     );
   });
 
+  async function setBalances(
+    receivers: string[],
+    amounts: number[],
+    totalSupply: number
+  ) {
+    expect(receivers.length).to.equal(amounts.length);
```

Comment:
I personally don't like having `expect` in helper functions. they don't check if a state is right 
also why don't you pass in an array of `{receiver: string, amount: number}`?

Reply:
Fixed in 08aa9e5c57d9581a9960d89c3329b258823f8637

Reply:
Improved in bbc468ded7f531d27ce685c55d3dd166acc20bef